phptravels website/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "settings": {
+    "base_url": "https://phptravels.net/login",
+    "implicit_wait_duration": 5
+  },
+  "xpaths": {
+    "login": {
+      "username_input": "//input[@type='email']",
+      "password_input": "//input[@type='password']",
+      "login_button": "//button[@class='btn btn-default btn-lg btn-block effect ladda-button waves-effect']"
+    },
+    "subpages": {
+      "hotels_sub": "//a[@class='active_hotels waves-effect']",
+      "city_box": "//span[@id='select2-hotels_city-container']",
+      "city_choice": "//ul[@id='select2-hotels_city-results']/li",
+      "room_choice": "//a[@data-toggle='dropdown']",
+      "room_inc": "//div[@class='roomInc']",
+      "submit_button": "//button[@id='submit']"
+    }
+  },
+  "credentials": {
+    "login": "user@phptravels.com",
+    "password": "demouser"
```

Comment:
better not to push credentials into repo. Try using environment variables and dotenv library.

Reply:
Changed